_docs-sources/iac/getting-started/deploying-a-module.md

```diff
@@ -144,8 +144,26 @@ Next, we’ll write a simple Python function that returns a string that will be
 Copy the following to `terraform-aws-gw-lambda-tutorial/main.py`.
 
 ```py title="terraform-aws-gw-lambda-tutorial/main.py"
+import json
+from urllib.request import urlopen, Request
```

Comment:
We might be able to simplify this using `requests`, which is an OSS package with a nicer interface than urllib. At one point was available in the python lambda runtime like so - `from botocore.vendored import requests`.

At one point AWS was considering removing it from the runtime, but it seems like they cancelled that work. Worth a try to reduce the number of lines of code?

Reply:
Very cool. Will give it a shot - I was indeed trying to keep this handler as minimal as possible - and hopefully the next pass I'll put up shortly will do just that!

Reply:
Alrighty - with the latest changes I can confirm everything works as intended end to end, and I think the resulting lambda handler code, while a bit longer, is not too overwhelming.

Reply:
excellent, sorry for the rabbit hole!